assets/react/v3/entries/addon-list/components/AddonCard.tsx

```diff
@@ -50,17 +50,28 @@ function AddonCard({ addon }: { addon: Addon }) {
       checked: checked,
     });
 
-    if (response.success) {
+    if (response.success || typeof response === 'string') {
       setUpdatedAddons([
-        ...updatedAddons.filter((item) => item.base_name === addon.base_name),
+        ...updatedAddons.filter((item) => item.basename !== addon.basename),
         { ...addon, is_enabled: checked ? 1 : 0 },
       ]);
+      showToast({
+        type: 'success',
+        message: checked ? __('Addon enabled successfully.') : __('Addon disabled  successfully.'),
```

Comment:
Text domain missing.

Also, we can use sprintf
```suggestion
        message: sprintf( __('Addon %s successfully.', 'tutor'), checked ? 'enabled' : 'disabled');
```